sdk/textanalytics/ai-text-analytics/src/analyzeSentimentResult.ts

```diff
@@ -41,24 +46,74 @@ export interface AnalyzeSentimentSuccessResult extends TextAnalyticsSuccessResul
   sentences: SentenceSentiment[];
 }
 
+/**
+ * The predicted sentiment for a given span of text. For more information regarding text sentiment, see https://docs.microsoft.com/azure/cognitive-services/Text-Analytics/how-tos/text-analytics-how-to-sentiment-analysis.
+ */
+export interface SentenceSentiment {
+  /**
+   * The sentence text.
+   */
+  text: string;
+  /**
+   * The predicted Sentiment for the sentence.
+   */
+  sentiment: SentenceSentimentLabel;
+  /**
+   * The sentiment confidence score between 0 and 1 for the sentence for all classes.
+   */
+  confidenceScores: SentimentConfidenceScores;
+
+  minedOpinions?: MinedOpinion[];
+}
+
+export interface AspectSentiment {
+  /**
```

Comment:
Doc string for the type needs to go before the interface declaration. The way this will be interpreted is as if it were documentation on the `confidenceScores` property.

Each property also needs its own doc string, which we can probably look and see what .NET and Python did on their pilot design and use the same/similar language.

Reply:
done.